nifi-nar-bundles/nifi-standard-services/nifi-ssl-context-service-api/src/main/java/org/apache/nifi/ssl/SSLContextService.java

```diff
@@ -109,10 +109,10 @@
          * Prepopulate protocols with generic instance types commonly used
          * see: http://docs.oracle.com/javase/7/docs/technotes/guides/security/StandardNames.html#SSLContext
          */
-        supportedProtocols.add("TLS");
+        supportedProtocols.add(TlsConfiguration.SSL_PROTOCOL);
 
         // This is still available for outgoing connections to legacy services, but can be disabled with jdk.tls.disabledAlgorithms
```

Comment:
Tiny issue here is that the comment refers to SSL still being available for outgoing connections but this switched the order so now it's implying TLS is still available for outgoing connections to legacy services.

Reply:
Otherwise LGTM +1

Reply:
Thanks for the review and catching this issue.  I pushed an update so that now the references match the existing comments.